### PR TITLE
ceph: resync crds

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

Add missing API fields.

Signed-off-by: Sébastien Han <seb@redhat.com>

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml [hunk 1]
```diff
@@ -356,3 +356,266 @@ spec:
   version: v1
   subresources:
     status: {}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (travisn) [inline cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml:360]: Should this file be included in a separate PR so we can backport the rest to 1.3?

Comment (leseb) [reply cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml:360]: The rest is already in 1.3 with https://github.com/rook/rook/pull/5780

Comment (travisn) [reply cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml:360]: of course!
